flang/test/Lower/OpenMP/omp-master.f90

```diff
@@ -0,0 +1,29 @@
+! This test checks lowering of OpenMP Master Directive to FIR Dialect.
+
+! RUN: bbc -fopenmp -emit-fir %s -o - | \
+! RUN:   FileCheck %s --check-prefix=FIRDialect
+
+program main
```

Comment:
We need to test more when we add lowering.

Reply:
This test case covers the basic yet useful scenario Master inside a Parallel region with an IO statement.
I'll try adding some cases with some conditional statements. Loops can be lowered to FIR dialect but cannot be lowered to LLVMIR Dialect.
If you have any specific cases in mind, could you please share.

Reply:
That is alright, I have nothing specific in mind. master in parallel captures the most important usage.
May be one where master is not immediately inside parallel. The parallel can be before a function/subroutine call and you could have the master only inside a function/subroutine.